k-distribution/tests/regression-new/set-symbolic-tests/intersect-23-spec.k.out

```diff
@@ -0,0 +1,16 @@
+  #Not ( {
+    Y
+  #Equals
+    Z
+  } )
+#And
+  <k>
+    intersectResult ( Z:MyId in SetItem ( Y:MyId ) ) ~> .
```

Comment:
Shouldn't this get simplified?

Reply:
No, unfortunately we do not have rule `E1 in (S SetItem(E2)) => E1 in S requlres E1 =/= E2` because it has severe performance impact.